x-pack/plugins/security_solution/public/flyout/right/tabs/table_tab.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { RightPanelContext } from '../context';
+import { TABLE_TAB_ERROR_TEST_ID, TABLE_TAB_CONTENT_TEST_ID } from './test_ids';
+import { TableTab } from './table_tab';
+import { TestProviders } from '../../../common/mock';
+
+const mockDispatch = jest.fn();
+jest.mock('react-redux', () => {
+  const original = jest.requireActual('react-redux');
+
+  return {
+    ...original,
+    useDispatch: () => mockDispatch,
+  };
+});
+
+describe('<TableTab />', () => {
+  it('should render table component', () => {
```

Comment:
Is it worth testing that an actual field shows up in the tab as well?

Reply:
I felt that it wasn't needed here as I'm using the existing `EventFieldsBrowser` component which already has some unit tests.